indolaw-parser/parser_utils.py

```diff
@@ -600,8 +600,13 @@ def insert_perubahan_section_open_quotes(law: List[str]) -> List[str]:
     for line in law:
         new_law.append(line)
 
+    i = 0
+    last_i = len(law)
+    # i_input contains a dicts of input index and the associated input
+    i_input = []
```

Comment:
could we name this something more explicit, maybe `user_input_history`? :) (the comment is good though 👍 )

Reply:
also instead of having the field be named `i`, would something like `line_index` be clearer? (this is my fault hehe for using `i` in the loop in the first place, but since it's outside the context of a loop feels even more important to be clear)

Reply:
also `user_input` instead of `input`?